src/base/bittorrent/torrentcreationmanager.cpp

```diff
@@ -2,6 +2,7 @@
  * Bittorrent Client using Qt and libtorrent.
  * Copyright (C) 2015, 2018  Vladimir Golovnev <glassez@yandex.ru>
  * Copyright (C) 2006  Christophe Dumez <chris@qbittorrent.org>
+ * Copyright (C) 2024  Radu Carpa <radu.carpa@cern.ch>
```

Comment:
Move all copyright strings to the top.

Reply:
I will fix it myself when I contribute to this PR.